README.md

```diff
@@ -57,17 +57,21 @@ Please refer to the release notes in [Releases](https://github.com/awslabs/amazo
 ### Download
 To download run the following command:
 
-`git clone --recursive https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c.git`
+```shell
+git clone --recursive https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c.git
+```
```

Comment:
Minor, but can we change this one to:
```
git clone --recursive https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c.git --single-branch -b main kvs-webrtc-sdk
```
- Recursive isn't necessary but it allows us to eventually add modules later when needed
- Single-branch (main) to speed up clone times (don't need all the other branches)
- kvs-webrtc-c-sdk to change the directory name (reduce directory length for some systems)

Reply:
Also could you remove the extra newline, unless it serves some additional purpose that I'm not aware of?

Reply:
* To avoid confusion, I'm against adding `--recursive` until we actually have submodules for it to do something with as it might inaccurately suggest that we do have submodules.
* Regarding cloning the single main branch, downloading the other branch indexes has insignificant effect on cloning times and repository size. Readily allowing customers and contributors to checkout any branch, on the other hand, I do see as a significant function to user experience.
  * Please see our discussion and my test results from a previous PR [here](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/pull/1173#:~:text=I%20tested%20the,checkout%20remote%20branches.).
* Changing the directory name to `kvs-webrtc-sdk` sounds good to me, but this might affect some of the scripts that we provide bundled in the SDK for customers to run. I will verify that there is no breaking functionalities with such a change.
* The new extra line for the closing  `` ``` `` is if for readability, similarly to how closing `}` in code typically appear on a new line.

Reply:
Ignore my comment about the newline, saw this and thought there was an extra line added:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/534da848-83e0-40e5-a313-fe1adba166aa">